cdk/domino_cdk/provisioners/efs.py

```diff
@@ -103,3 +107,19 @@ def provision_backup_vault(self, stack_name: str, efs_backup: config.EFS.Backup)
             backup_selection_name=f"{stack_name}-efs",
             role=backupRole,
         )
+
+        DominoCdkUtil.create_lambda(
+            scope=self.scope,
+            stack_name=stack_name,
+            dirname=path.dirname(path.abspath(__file__)),
+            name="backup_post_creation_tasks",
+            environment={"stack_name": stack_name, "backup_vault": vault.backup_vault_name},
+            resources=[
+                "arn:aws:logs:*",
+                f"arn:aws:backup:{self.scope.region}:{self.scope.account}:backup-vault/{stack_name}-efs*",
```

Comment:
Please make sure the partition uses the scope or use the `Arn` helper so this works in govcloud, e.g.
```
f"arn:{partition}:...
```
or 
https://github.com/dominodatalab/cdk-cf-eks/blob/89dc604e80e5768f034c576553ce21ad55179fd5/cdk/domino_cdk/provisioners/eks/eks_iam.py#L58

Reply:
Will do

Reply:
This too?

https://github.com/dominodatalab/cdk-cf-eks/blob/89dc604e80e5768f034c576553ce21ad55179fd5/cdk/domino_cdk/provisioners/eks/eks_iam_roles_for_k8s.py#L140